app/bundles/EmailBundle/Controller/Api/EmailApiController.php

```diff
@@ -214,4 +215,13 @@ public function replyAction(Reply $replyService, RandomHelperInterface $randomHe
             $this->view(['success' => true], Response::HTTP_CREATED)
         );
     }
+
+    protected function prepareParametersFromRequest(FormInterface $form, array &$params, object $entity = null, array $masks = [], array $fields = []): void
+    {
+        if (isset($params['publicPreview']) && $entity instanceof Email) {
+            $entity->setPublicPreview(InputHelper::boolean($params['publicPreview']) ?? false);
```

Comment:
Didn't you mean this instead?
```suggestion
            $entity->setPublicPreview(InputHelper::boolean($params['publicPreview'] ?? false));
```

Reply:
@escopecz good catch

Reply:
I am gonna revert the commit, Based on code, isset($params['publicPreview']) is in parent condition and this InputHelper::booleen in some cases can return null. Then it's corret, always set bool in setPublicPreview